pom.xml

```diff
@@ -370,9 +376,9 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <skipTests>false</skipTests>
+        <skipTests>true</skipTests>
```

Comment:
What is this option doing?

Reply:
This is skipping spotbugs and pmd because otherwise it won't let me run `mvn package`
Will have to be switched back to false before deployment.

Reply:
Ok. Make sure there is a SIM for it.

Reply:
Rolled back - can just run with -DskipTests to achieve same thing